app/controllers/campaigns_controller.rb

```diff
@@ -1,54 +1,53 @@
 class CampaignsController < ApplicationController
-  # before_action :authenticate_user!
+  before_action :authenticate_user!
 
- def index
-    @campaigns = Campaign.all
- end
+  def index
+    @campaigns = current_user.campaigns.all
```

Comment:
Remove the .all here

Reply:
I already made that correction in the testing branch

Reply:
Is it not the .all I’m still seeing here? Just current_user.campaigns is enough

Reply:
oh i thought u meant the Campaign.all should be change to current_user.campaigns.all. working on it now asap